test/event_handler_test.py

```diff
@@ -0,0 +1,100 @@
+import os
+import boto3
+import io
+import json
+import uuid
+from moto import mock_sqs
+from unittest import TestCase
+from unittest.mock import patch
+
+from src import event_handler
+
+
+@mock_sqs
+class EventHandlerTest(TestCase):
+
+    def setUp(self):
+        setup_stub_aws_config()
+        self.__sqs_client = boto3.client('sqs')
+        self.__queue_url = create_queue(self.__sqs_client)
+        add_queue_url_to_environment_variables(self.__queue_url)
+
+    @patch('sys.stdout', new_callable=io.StringIO)
+    def test_reads_messages_from_queue(self, mock_stdout):
+        self.__send_messages_to_queue(
+            [
+                create_event_string('sample-id-1'),
+                create_event_string('sample-id-2'),
+            ]
+        )
+
+        event_handler.store_queued_events(None, None)
+
+        self.assertEqual(mock_stdout.getvalue(), 'sample-id-1\nsample-id-2\n')
+        self.assertEqual(self.__number_of_visible_messages(), '0')
+        self.assertEqual(self.__number_of_hidden_messages(), '0')
+
+    @patch('sys.stdout', new_callable=io.StringIO)
+    def test_does_not_delete_invalid_messages(self, mock_stdout):
+        self.__send_messages_to_queue(
```

Comment:
Something to discuss - should we be moving messages to a dead letter queue?  Does SQS do this somehow automatically?  (not a blocker for this PR!)

Reply:
Yes, SQS does this for us, which is a nice separation of concerns.

SQS keeps track of how many times a message has become visible, and you can configure SQS with a 'max retry count' (actual name might be slightly different). You can configure behaviour on reaching this count, including sending to DLQ